src/build.rs

```diff
@@ -446,11 +446,26 @@ impl BuildQueue {
             let mut manifest_path = build_dir.clone();
             manifest_path.push("Cargo.toml");
             trace!("manifest_path: {:?}", manifest_path);
+            // TODO: Add support for virtual manifests and multiple packages 
             let ws = Workspace::new(&manifest_path, &config).expect("could not create cargo workspace");
+            let current_package = ws.current().unwrap(); 
+            let targets = current_package.targets(); 
+
+            let bins = {
+                if rls_config.build_bin.is_empty() { 
+                    vec![]
+                } else {
+                    let mut bins = targets.iter().filter(|x| x.is_bin()); 
+                    let bin = bins.find(|x| x.name() == rls_config.build_bin);
+                    match bin { None => vec![], Some(bin) => vec![bin.name().to_owned()]}
```

Comment:
Oh, right! While I worked on this, I implemented also returning notifications from build phase and passing them to the user, but I thought I can't use return any meaningful message for now (e.g. for implicit/detected targets by cargo), so I just separated that and didn't include it. But we definitely can here, as bin target has to be explicitly named in the manifest file! The notification bit is [here](https://github.com/Xanewok/rls/commit/8928af24685101880a62aa1c5fcca9944c147280), I can push PR with that later. For now is `debug!` okay?